### DISCUSSION TITLE
query columns information fail,  exception:  failed to call frontend service/n

### DISCUSSION THREAD

**Discussion by cjd2223** [apache/doris#15265] [A - General / Q&A]
doris-version : doris-1.2.0-rc04-Unknown

Log in FE, query column metadata information. 
SQL statement is as follows:
**select * from information_schema.columns;**

The query fails. The following exceptions are displayed:
`java.sql.SQLException: (conn=7) RpcException, msg: org.apache.doris.rpc.RpcException: failed to call frontend service/n @ 0x556a2178c0aa doris::Status::ConstructErrorStatus() @ 0x556a222ff419 doris::ThriftRpcHelper::rpc<>() @ 0x556a21f53057 doris::SchemaHelper::describe_table() @ 0x556a21f55846 doris::SchemaColumnsScanner::get_new_desc() @ 0x556a21f58391 doris::SchemaColumnsScanner::get_next_row() @ 0x556a23285f93 doris::vectorized::VSchemaScanNode::get_next() @ 0x556a21d80e26 doris::ExecNode::get_next_after_projects() @ 0x556a220a0de3 doris::PlanFragmentExecutor::get_vectorized_internal() @ 0x556a220a50c8 doris::PlanFragmentExecutor::open_vectorized_internal() @ 0x556a220a6a25 doris::PlanFragmentExecutor::open() @ 0x556a2207edbc doris::FragmentExecState::execute() @ 0x556a220821eb doris::FragmentMgr::_exec_actual() @ 0x556a2208288a _ZNSt17_Function_handlerIFvvEZN5doris11FragmentMgr18exec_plan_fragmentERKNS1_23TExecPlanFragmentParamsESt8functionIFvPNS1_20PlanFragmentExecutorEEEEUlvE_E9_M_invokeERKSt9_Any_data @ 0x556a2232d815 doris::ThreadPool::dispatch_thread() @ 0x556a22323bff doris::Thread::supervise_thread() @ 0x7fccf60a6ea5 start_thread @ 0x7fccf63b9b0d __clone @ (nil) (unknown)`

**Comment by JNSimba** [comment]
The operation here is ok, you can check whether your fe/be status is normal through `show frontends` and `show backends`

**Reply by cjd2223** [reply]
The fe.`Alive` is true.
The be.`Alive` is also true.
I installed a new doris on a different server and it's fine.
Thanks for your reply.
